hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -418,13 +418,13 @@ spec = parallel $ do
               withHydraNode bobSk [alice] chain $ \n2 -> do
                 openHead chain n1 n2
                 let decommitTx1 = SimpleTx 1 (utxoRef 1) (utxoRef 42)
-                send n2 (Decommit{decommitTx = decommitTx1})
+                send n1 (Decommit{decommitTx = decommitTx1})
```

Comment:
why do we swap n1 for n2 in this spec?
should not it remain the same?

Reply:
It is because n1 is actually the owner of `utxoRef 1` which is to be decommited here.